basictracer/text_propagator.py

```diff
@@ -37,9 +37,9 @@ def extract(self, carrier):  # noqa
                 trace_id = int(v, 16)
                 count += 1
             elif k == field_name_sampled:
-                if v == str(True).lower():
+                if v in (str(True).lower(), '1'):
```

Comment:
is `str(True).lower()` better than `true` ? :-)

Reply:
lol :D ask mr. Brandon Gonzalez (git blame) :D I'm going to send better expression, w8 pls.